Makefile

```diff
@@ -2,18 +2,14 @@
 
 HTMLCOV_DIR ?= htmlcov
 
-test: flake8 test_lib
+test: flake8 pylint pytest
```

Comment:
just curious why you use both flake8 and pylint? the other projects just seem to use flake8?